packages/admin-next/dashboard/src/components/layout/notifications/notifications.tsx

```diff
@@ -68,8 +69,9 @@ export const Notifications = () => {
           >
             responseKey="notifications"
             queryKey={notificationQueryKeys.all}
-            queryFn={(params) => sdk.admin.notification.list(params)}
+            queryFn={(params) => sdk.admin.notification.list({ id: "test" })}
```

Comment:
todo: This param looks incorrect
```suggestion
            queryFn={(params) => sdk.admin.notification.list()}
```

Reply:
ah, damn 😄